### PR TITLE
Add feature state templates

### PR DESCRIPTION
**Pull request by chenopis**
Add templates to mark features as `alpha`, `beta`, or `stable`.

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3680)

### CODE REVIEW

File: _includes/feature-state-alpha.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% assign dialog_title = "alpha" %}
+{% capture dialog_content %}
+This feature is currently in a `alpha` state, meaning:
+
+* The version names contain alpha (e.g. v1alpha1).
+* May be buggy. Enabling the feature may expose bugs. Disabled by default.
```
Comment (steveperry-53) [inline _includes/feature-state-alpha.md]: may -> might (Several places)

File: _includes/feature-state-beta.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% assign dialog_title = "beta" %}
+{% capture dialog_content %}
+This feature is currently in a `beta` state, meaning:
+
+* The version names contain beta (e.g. v2beta3).
+* Code is well tested. Enabling the feature is considered safe. Enabled by default.
+* Support for the overall feature will not be dropped, though details may change.
+* The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release. When this happens, we will provide instructions for migrating to the next version. This may require deleting, editing, and re-creating API objects. The editing process may require some thought. This may require downtime for applications that rely on the feature.
+* Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases. If you have multiple clusters which can be upgraded independently, you may be able to relax this restriction.
```
Comment (steveperry-53) [inline _includes/feature-state-beta.md]: which -> that

File: _includes/feature-state-beta.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+* **Please do try our beta features and give feedback on them! Once they exit beta, it may not be practical for us to make more changes.**
```
Comment (steveperry-53) [inline _includes/feature-state-beta.md]: Once -> After

File: docs/concepts/workloads/pods/init-containers.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ scripts not present in an app image.
 
 {:toc}
 
-{% include 1-5-beta.md %}
+{% assign for_k8s_version="v1.5" %}{% include feature-state-beta.md %}
 
 **Once the feature exits beta, Init Containers will be specified in the PodSpec
```
Comment (steveperry-53) [inline docs/concepts/workloads/pods/init-containers.md]: Our style is not to use bold for important text. Instead, we separate out the important text as a **Note** , **Warning**, **Important**, etc.

File: _includes/feature-state-alpha.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{% assign dialog_title = "alpha" %}
+{% capture dialog_content %}
+This feature is currently in a `alpha` state, meaning:
```
Comment (steveperry-53) [inline _includes/feature-state-alpha.md]: Is 'alpha' from the code somewhere? Or are you just using back ticks for emphasis and to introduce a term? If it's for introducing a term, then we should use italic.

File: docs/concepts/workloads/pods/init-containers.md (hunk 4 above)
Comment (chenopis) [inline docs/concepts/workloads/pods/init-containers.md]: Are you referring to the **FEATURE STATE** tag from the include or the additional text, "**Once the feature exits beta**..."? The latter was there from before, but I can change that too while I'm at it.

File: _includes/feature-state-alpha.md (hunk 5 above)
Comment (chenopis) [inline _includes/feature-state-alpha.md]: Okay, I'll switch those to italics.

File: _includes/feature-state-alpha.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+* The version names contain alpha (e.g. v1alpha1).
+* May be buggy. Enabling the feature may expose bugs. Disabled by default.
+* Support for feature may be dropped at any time without notice.
```
Comment (steveperry-53) [inline _includes/feature-state-alpha.md]: I can't vouch for the technical accuracy of these descriptions, so we'll need a tech review. For example, I could swear I've used alpha features that were enabled by default. At least I don't remember explicitly enabling them.

**Comment by steveperry-53** [comment]
@chenopis, Thanks, this looks great. Some comments in the files.

File: _includes/feature-state-alpha.md (hunk 6 above)
Comment (chenopis) [inline _includes/feature-state-alpha.md]: I pulled the API versioning copy from https://kubernetes.io/docs/reference/api-overview/, which was left there from before.

File: docs/concepts/workloads/pods/init-containers.md (hunk 4 above)
Comment (steveperry-53) [inline docs/concepts/workloads/pods/init-containers.md]: I was referring to "Once the feature exits." Another possibility is to use a background shade or color. Example: https://cloud.google.com/container-registry/docs/

**Pull request merged**